content/en/opentelemetry/collector_exporter/collector_batch_memory.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Batch and Memory Settings
+further_reading:
+- link: "/opentelemetry/collector_exporter/"
+  tag: "Documentation"
+  text: "Getting Started with Collector"
+---
+
+## Overview
+
+To edit your OpenTelemetry Collector batch and memory settings, configure the [batch processor][1] in your Datadog Exporter.
+
+For more information, see the OpenTelemetry project documentation for the [batch processor][1].
+
+## Setup
+
+{{< tabs >}}
+{{% tab "Host" %}}
+
+```yaml
+processors:
+  batch:
+    # Datadog APM Intake limit is 3.2MB.    
+    send_batch_max_size: 1000
+    send_batch_size: 100
+    timeout: 10s
+  memory_limiter:
+    check_interval: 1s
+    limit_mib: 1000
+```
+
+{{% /tab %}}
+
+{{% tab "Kubernetes" %}}
+
+Add the following lines to `values.yaml`:
+
+```yaml
+resources:
+  limits:
+    cpu: 512m
+    memory: 1Gi
+```
+
+Collector configuration:
+
+```yaml
+processors:
+  batch:
+    # Datadog APM Intake limit is 3.2MB.    
+    send_batch_max_size: 1000
+    send_batch_size: 100
+    timeout: 10s
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+```yaml
```

Comment:
I don't think this prometheus receiver should be here. The prometheus receiver is for the Collector Health Metrics sections